mcgill-en.csl

```diff
@@ -17,14 +17,17 @@
       <email>f.martin-bariteau@umontreal.ca</email>
       <uri>http://f-mb.github.io/cslegal/</uri>
     </contributor>
+			  
```

Comment:
did something go wrong here? No reason to space these out otherwise.

Reply:
Hi, and yes.  I will remove the spaces.